.github/workflows/post_check_svgs_comment.yml

```diff
@@ -54,7 +53,7 @@ jobs:
 
       - name: Comment on the PR about the result - SVG Error
         uses: jungwinter/comment@v1 # let us comment on a specific PR
-        if: success() && steps.err_message_reader.outputs.content != 'None'
+        if: success() && (steps.err_message_reader.outputs.content != '0' || steps.err_message_reader.outputs.content != '1')
```

Comment:
I'm not 100% sure why, but this condition seems not to work since the bot is now posting a message even when the content is 0. 

Also I suggest to reuse the enum here:
```suggestion
        if: success() && (steps.err_message_reader.outputs.content != SVG_STATUS_CODE.NO_SVG.value || steps.err_message_reader.outputs.content != SVG_STATUS_CODE.SVG_OK.value)
```

Reply:
The reason for content is '0' but the script fails is due to the workflow-run restriction. 

![image](https://user-images.githubusercontent.com/43018778/108745139-98883080-74ef-11eb-8c2f-5bbec8991d29.png)


The change will only take effect when the yaml file is merged into master. You can test this by creating a new step in the workflow file like this:

```
- name: Test
  run: echo hello world
```

Then check the Actions tab. The new step will not show up even though the syntax is valid.

Reply:
Thanks for the explanation!